app/views/account/show.njk

```diff
@@ -0,0 +1,60 @@
+{% extends "layouts/auth.njk" %}
+
+{% set headerNavId = "account" %}
+
+{% set hidePrimaryNavigation = true %}
+{% set hideOrganisationSwitcher = true %}
+
+{% set title = "Your account" %}
```

Comment:
Your or my account? 😄

Reply:
Your account is correct. I've updated the PR title.